src/MOI_wrapper/MOI_wrapper.jl

```diff
@@ -3497,6 +3499,7 @@ function MOI.modify(
     ::MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}},
     chg::MOI.ScalarCoefficientChange{Float64},
 )
+    _update_if_necessary(model)
```

Comment:
These ones didn't seem to be necessary for the test, but I added just in case. I don't understand the internal details.